lib/util/memcache.ex

```diff
@@ -6,10 +6,50 @@ defmodule Antikythera.Memcache do
   @moduledoc """
   In-memory cache for each executor pool.
 
-      iex> Antikythera.Memcache.write("foo", "bar", epool_id)
+  Antikythera provides in-memory cache controlled by AntikytheraCore for each executor pool.
+  Objects are stored in ETS internally.
+
+  ## Usage
+
+  Memcache behaves as a key-value storage.
+
+      iex> Antikythera.Memcache.write("foo", "bar", epool_id, 3_600)
       :ok
       iex> Antikythera.Memcache.read("foo", epool_id)
       {:ok, "bar"}
+
+  ## Limitation
+
+  The number of records and the size of keys and values is limited.
+
+  - The maximum number of records for each executor pool is #{AntikytheraCore.ExecutorPool.MemcacheWriter.max_records()}.
+    - If exceeds the limit, records whose expiration time is nearest are evicted.
```

Comment:
Please use 4-space indents in markdown sub-lists.

(I couldn't just find the canonical spec though. At least we are uniformly using 4-space.)

Reply:
I've been advocating the 4-space indentation for markdown lists; you guys should stick with it :)

The reason is simple: layout of bullet/numbered list should reflect its structure even with multi-line items.
- Good (structure of list is visually obvious)
  ```
  - Long text 1 ...
    continued ...
  - Long text 2 ...
    continued ...
  ```
  Bad
  ```
  - Long text 1
  continued ...
  - Long text 2 ...
  continued ...
  ```
- Good
  ```
  - Long text 1 ...
    continued ...
      - Sub item A
      - Sub item B
  ```
  Bad (lines in different levels start at the same position)
  ```
  - Long text 1 ...
    continued ...
    - Sub item A
    - Sub item B
  ```

Reply:
Indeed, it's so reasonable to use 4-space indentation!

Fixed in e8adcb50f657f23724421e7b0383d11d95fe9bb0